examples/complete/version.tf

```diff
@@ -4,7 +4,7 @@ terraform {
     # Pin to the lowest provider version of the range defined in the main module to ensure lowest version still works
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.51.0"
+      version = ">= 1.51.0"
```

Comment:
since you have to update the range constraint here, remove the comment on line 4